base_layer/common_types/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub mod emoji;
 pub mod encryption;
 pub mod epoch;
 pub mod grpc_authentication;
+pub mod serde_with;
```

Comment:
```suggestion
pub mod serializers;
```
There is a popular crate called `serde_with` and reusing this name will be misleading